chatbot/generate_chatbot.py

```diff
@@ -0,0 +1,63 @@
+## import libraries
+import os
+from vectordb import ChatbotAgent
+from load_data import load_data
+
+
+
+# main function
+def main():
```

Comment:
I think you don't need to have such logic in the book repo right? If I understand correctly, this should be launched in our server side?

Reply:
You have the reason, but for now, it runs locally, so it is better to define a main function. But if we deploy our model to AWS lambda, I think it won't have much effect on that. For now, I think it is better to keep it before launching.